load-tests/h2_h1c_passthrough/scripts/load_test/main.bal

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Cant we use default jmeter instead of the ballerina script?

Reply:
AFAIK we cannot test HTTP2 services using jmeter.